src/patterns/step-by-step-navigation/index.md.njk

```diff
@@ -61,6 +61,8 @@ You can use the following examples in the GOV.UK Prototype Kit to prototype a st
 - [Step by step page](https://govuk-prototype-kit.herokuapp.com/docs/templates/step-by-step-navigation)
 - [Start page with step by step navigation as a sidebar](https://govuk-prototype-kit.herokuapp.com/docs/templates/start-with-step-by-step)
 
+Remember that step by step navigation is only for use on GOV.UK. We have included it in the Prototype Kit only so you can prototype your end to end journeys. We do not provide the code for step by step navigation, unlike most other components and patterns in the Design System.
```

Comment:
This read slightly strangely to me, as you _do_ provide the code for the pattern, but only in the prototyping kit, and not in `govuk-frontend`.

```suggestion
Remember that step by step navigation is only for use on GOV.UK. We have included it in the Prototype Kit only so you can prototype your end to end journeys. We do not provide the code for step by step navigation in `govuk-frontend`, unlike most other components and patterns in the Design System.
```

Reply:
Ah gotcha - thanks, @frankieroberto! Would just rejig the last sentence slightly, to avoid suggesting that `govuk-frontend`= a component or pattern.

```suggestion
Remember that step by step navigation is only for use on GOV.UK. We have included it in the Prototype Kit only so you can prototype your end to end journeys. Unlike most other components and patterns in the Design System, we do not provide the code for step by step navigation in `govuk-frontend`.
```

Reply:
Making this suggestion in case what GOV.UK is, is unclear to some users. ```suggestion
Remember that step by step navigation is not for use within services. We have included it in the Prototype Kit only so you can prototype your end to end journeys. We do not provide the code for step by step navigation, unlike most other components and patterns in the Design System.
```

Reply:
@EoinShaughnessy nitpicking, but does this imply that "GOV.UK" is not a service (which it arguably is)?  I suspect that the GOV.UK vs services distinction is lost on a lot of people, understandably.

Could we say "not for use within transactional services"? This also acknowledges that other non-central-government websites might use it (eg devolved administrations or local authorities).

Reply:
@frankieroberto Not nitpicking at all! That's a really good suggestion, thanks for explaining the rationale.